sdk/cosmosdb/cosmos/src/ChangeFeedIterator.ts

```diff
@@ -1,16 +1,16 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 /// <reference lib="esnext.asynciterable" />
-import type { ChangeFeedOptions } from "./ChangeFeedOptions";
-import { ChangeFeedResponse } from "./ChangeFeedResponse";
-import type { Resource } from "./client";
-import type { ClientContext } from "./ClientContext";
-import { Constants, ResourceType, StatusCodes } from "./common";
-import type { DiagnosticNodeInternal } from "./diagnostics/DiagnosticNodeInternal";
-import type { PartitionKey } from "./documents";
-import type { FeedOptions } from "./request";
-import type { Response } from "./request";
-import { getEmptyCosmosDiagnostics, withDiagnostics } from "./utils/diagnostics";
+import type { ChangeFeedOptions } from "./ChangeFeedOptions.js";
```

Comment:
I don't see .js file in the parent folder, how does this works?

Reply:
It's the combination of [mandatory file extensions](https://nodejs.org/api/esm.html#mandatory-file-extensions) in Node, and what TypeScript does to enforce this by changing .ts to .js.